### PR TITLE
Adding the organisation field to users object

### PR DESCRIPTION
**Pull request by LanternNassi**
# Description

Adding the organisation field to the users creation process

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Trello Ticket ID

https://trello.com/c/draQuk5k/1521-add-the-organisation-field-to-users-information

## How Can This Be Tested?

Migrate the changes to your database first and then try creating a new user

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/models/user.py [hunk 1]
```diff
@@ -27,16 +27,18 @@ class User(ModelMixin):
...
     last_seen = db.Column(db.DateTime, default=db.func.current_timestamp())
     projects = db.relationship('Project', backref='owner', lazy=True)
+    organisation = db.Column(db.String(256), nullable=True, default="")
     other_projects = db.relationship('ProjectUser', back_populates='user')
     is_beta_user = db.Column(db.Boolean, nullable=False, default=False)
     credits = db.relationship('Credit', backref='user', lazy=True)
     credit_assignments = db.relationship('CreditAssignment', backref='user', lazy=True)
 
-    def __init__(self, email, name, password):
+    def __init__(self, email, name, password, organisation):
```
Comment (MutegekiHenry) [inline app/models/user.py:36]: @LanternNassi this line makes the tests fail as the organisation field is not reflected/updated in the test cases. So you can resolve this by adding the organization field in the positive test case with an example value See error below 

Screenshot 2023-06-21 at 11 33 26

File: app/schemas/user.py [hunk 2]
```diff
@@ -29,7 +29,14 @@ class UserSchema(Schema):
     age = fields.Method("get_age", dump_only=True)
     is_beta_user = fields.Boolean()
     credits = fields.Nested(CreditSchema, many=True, dump_only=True)
-
+    organisation = fields.String(required=True, error_message={
```
Comment (MutegekiHenry) [inline app/schemas/user.py:32]: This was made nullable in the database, yet we have it as required here, lets have a consistent constraint

**Review by MutegekiHenry — CHANGES_REQUESTED** [review]
Hi @LanternNassi great work on this. Just a few comments to address

**Comment by jnsereko** [comment]
you  can also try and fix the failing builds @LanternNassi

**Review by MutegekiHenry — APPROVED** [review]
LGTM

**Pull request merged**